src/js/jsx/sections/libraries/LibraryBar.jsx

```diff
@@ -117,6 +119,38 @@ define(function (require, exports, module) {
                     <Gutter />
                 </div>
             );
+        },
+        
+        _createColorButton: function (attr, buttonText) {
+            var layer = this.props.selectedLayers.first();
+            if (this.props.selectedLayers.size !== 1 || layer[attr].isEmpty()) {
+                return null;
+            }
+            
+            var color = layer[attr].first().color;
+            return (
+                <SplitButtonItem title={buttonText} disabled={this.props.disabled}>
+                    <div className="split-button__item__color-icon"
+                         style={{ backgroundColor: color.toCssRGB() }}
+                         onClick={this.addColorAsset.bind(this, color)} />
+                </SplitButtonItem>
+            );
+        },
+        
+        _canAddGraphic: function () {
```

Comment:
The only exception we have is for the React lifecycle methods (`render`, et al.) If no additional discussion is required, it's sufficient to annotate it as `@private` as well as the return and/or parameter types.